cmake/process_users_input.cmake

```diff
@@ -21,20 +21,31 @@ set_property(CACHE ORDER PROPERTY STRINGS ${ORDER_OPTIONS})
 
 set(NUMBER_OF_MECHANISMS 0 CACHE STRING "Number of mechanisms")
 
-
 set(EQUATIONS "elastic" CACHE STRING "Equation set used")
 set(EQUATIONS_OPTIONS elastic anisotropic viscoelastic viscoelastic2)
 set_property(CACHE EQUATIONS PROPERTY STRINGS ${EQUATIONS_OPTIONS})
 
 
-set(ARCH "hsw" CACHE STRING "Type of the target architecture")
-set(ARCH_OPTIONS noarch wsm snb hsw knc knl skx thunderx2t99)
-set(ARCH_ALIGNMENT   16  16  32  32  64  64  64 16)  # size of a vector registers in bytes for a given architecture
-set_property(CACHE ARCH PROPERTY STRINGS ${ARCH_OPTIONS})
+set(HOST_ARCH "hsw" CACHE STRING "Type of the target host architecture")
+set(HOST_ARCH_OPTIONS noarch wsm snb hsw knc knl skx thunderx2t99 power9)
+# size of a vector registers in bytes for a given architecture
+set(HOST_ARCH_ALIGNMENT   16  16  32  32  64  64  64           16     16)
```

Comment:
Spacing is really strange here

Reply:
But it is easy to understand which alignment belongs to which architecture. It is done on purpose